lib/ui/src/chat/components/EnhancedContext.tsx

```diff
@@ -36,35 +36,21 @@ export const EnhancedContext: React.FunctionComponent<{
         return
     }
 
-    const uniqueFiles = new Set<string /* uri.toString() */>()
-
-    const filteredFiles = contextFiles.filter(file => {
-        if (uniqueFiles.has(file.uri.toString())) {
-            return false
-        }
-        uniqueFiles.add(file.uri.toString())
-        return true
-    })
-
-    if (!filteredFiles.length) {
-        return
-    }
-
     // Enhanced Context are context added by one of Cody's context fetchers.
     // NOTE: sparkle should only be added to messages that use enhanced context.
     // NOTE: Core chat commands (e.g. /explain and /smell) use local context only.
     // Check if the filteredFiles only contain local context (non-enhanced context).
     const localContextType = ['user', 'selection', 'terminal', 'editor']
-    const localContextOnly = filteredFiles.every(file => localContextType.includes(file.type))
+    const localContextOnly = contextFiles.every(file => localContextType.includes(file.type))
     const sparkle = localContextOnly || isCommand ? '' : '✨ '
     const prefix = sparkle + 'Context: '
     // It checks if file.range exists first before accessing start and end.
     // If range doesn't exist, it adds 0 lines for that file.
-    const lineCount = filteredFiles.reduce(
+    const lineCount = contextFiles.reduce(
         (total, file) => total + (file.range ? file.range?.end?.line - file.range?.start?.line + 1 : 0),
         0
     )
-    const fileCount = filteredFiles.length
+    const fileCount = contextFiles.length
```

Comment:
This still needs to be deduped. Something like this:

```suggestion
    const fileCount = new Set(contextFiles.map(file => file.uri.toString())).size
```